format/src/main/thrift/schema.thrift

```diff
@@ -33,6 +33,7 @@ enum DataType {
 	TIMESTAMP = 6,
 	DATE = 7,
 	BOOLEAN = 8,
+	BINARY = 19,
```

Comment:
Suggest to add it in the last and use value 26, not sure why this enum is not continuous.

Reply:
done